package.json

```
@@ -34,6 +34,7 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-prettier": "^5.1.3",
     "prettier": "^3.2.5",
+    "react-lottie-player": "^2.0.0",
```

```suggestion
    "react-lottie-player": "^2.0.0",
```
P5: 로그인 로딩 페이지 하나 하려고 라이브러리 설치하기는 살짝 아쉬운 감이 있는데 css 애니메이션 같은걸로 대체할 방법이 없을까요??

---

css로 한 번 해보겠습니다

---

상자 움직이는건 css로 구현을 하긴 했는데, 아래 글씨 부분이 너무 어렵네요...ㅠㅠ

---

일단 다른 파트 먼저 구현하고 다시 돌아올게여...ㅠㅠ 생각보다 시간이 좀 걸릴 것 같습니다...